### DISCUSSION TITLE
Packet to send air bubble count to players?

### DISCUSSION THREAD

**Discussion by ryleu** [Minestom/Minestom#1148] [Help] [answered]
What packet should I send to players to tell them how much air they have left? I saw that I need to track it myself, per #912, but it didn't say what packet class I need to use and I couldn't find it [in the Javadocs](https://javadoc.minestom.net/net/minestom/server/network/packet/server/SendablePacket.html) by `ctrl + f`ing for "air". Is there a pre-made packet for it, or do I need to make my own?

**Comment by MrlnHi** [comment, marked as answer]
Entity Metadata packet
https://c4k3.github.io/wiki.vg/Protocol.html#Entity_Metadata

**Comment by ryleu** [comment]
@Marlonzockt's led me to find this:
```kt
player.entityMeta.airTicks
```
I just have to set that and it works. Thanks!
